script/openqa-bootstrap

```diff
@@ -5,6 +5,18 @@ dbname="${dbname:="openqa"}"
 dbuser="${dbuser:="geekotest"}"
 
 # add extra repos for leap
+foundrepo() {
+	zypper lr "$1" && return
+	return 1
+}
+
+source /etc/os-release
+if [[ "$NAME" == "openSUSE Leap"]] ; then
+	if ! foundrepo devel:openQA; then zypper -n addrepo -p 90 obs://devel:openQA devel:openQA; fi
+	if ! foundrepo devel:openQA:Leap:${VERSION}; then zypper -n addrepo -p 91 obs://devel:openQA:Leap:${VERSION} devel:openQA:Leap:${VERSION}; fi
+	zypper -n  --gpg-auto-import-keys refresh
+fi
+
 source /etc/os-release
```

Comment:
I guess you should rather replace the block further below, and not add here on top.